src/Tests/DIPS.Xamarin.UI.Tests/Converters/ValueConverters/DateAndTimeConverterTests.cs

```diff
@@ -46,7 +46,7 @@ public void Convert_NullInput_ShouldReturnEmptyString()
                 new object[] { "en", new DateTime(1991, 12, 12,10,12,12), "12th Dec 1991 10:12 AM" },
             };
 
-        [Theory]
+        [Theory(Skip="Issue with region")]

```

Comment:
Remove skip, this will be fixed when this is merged: https://github.com/DIPSAS/DIPS.Xamarin.UI/pull/208

Reply:
Done